linux_docker_resources/entry_point.sh

```diff
@@ -32,7 +32,7 @@ case "${CI_ARGS}" in
         ;;
       *)
         echo "Installing Connext binaries off RTI website..."
-        python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti_connext_dds-5.3.0-eval-x64Linux3gcc5.4.0.run /home/rosbuild
+        python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti_connext_dds-5.3.0-eval-x64Linux3gcc5.4.0.run /home/rosbuild/rti_connext_dds-5.3.0 --rtipkg_paths /tmp/rti_security_plugins-5.3.0-eval-x64Linux3gcc5.4.0.rtipkg
```

Comment:
Why does the `install_directory` change?

Reply:
Right now the new function takes the install location and append `/bin/rtipkginstall`.
We could work around it in different ways but this seems to be the least invasive and simpler approach.

Current approach: path the full installation directory given that we know what it is based on the installer we use
Alternatives:
- add another argument to the script to pass the installation folder name
- guess it based on the installer filename we pass to the script
- hardcode it in the script

Reply:
Before the code was passing `/home/rosbuild` which made `install_connext()` install into `/home/rosbuild/rti_connext_dds-5.3.0`.

If this patch passes `/home/rosbuild/rti_connext_dds-5.3.0` doesn't the function install Connext into `/home/rosbuild/rti_connext_dds-5.3.0/rti_connext_dds-5.3.0`?

Reply:
The file location after installation is the same if you pass `/home/rosbuild` or `/home/rosbuild/rti_connext_dds-5.3.0`

CI passing with the current patch: http://ci.ros2.org/job/ci_linux/3316 (ignore the security test failures they're due to an unrelated domain_id issue)

Reply:
> The file location after installation is the same if you pass `/home/rosbuild` or `/home/rosbuild/rti_connext_dds-5.3.0`


Interesting... Sounds good to me then.